app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt

```diff
@@ -118,12 +118,20 @@ class ProfileListFragmentTest {
         )
       )
       onView(
-        atPositionOnView(R.id.profile_list_recycler_view, 0, R.id.profile_list_name)
+        atPositionOnView(
+          recylerViewId = R.id.profile_list_recycler_view,
```

Comment:
A typo here, similarly for all the below arguments 

```suggestion
          recyclerViewId = R.id.profile_list_recycler_view,
```

Reply:
Ok sir , Will do that, And Could you tekk mw how could I revert commit changes in my Pr, I know it locally but don't know about how to do it in a PR.

Reply:
@FareesHussain  Are changes fine now Sir?